### PR TITLE
Blog update

### PR DESCRIPTION
**Pull request by hacktivist123**
added external blog posts

### CODE REVIEW

File: src/posts/2023-07-03-ama-bill/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-07-03T11:28:00.000Z'
+title: 'Ask me anything with Bill Mulligan from Isovalent'
+ogImage: ogimage.jpeg
+ogSummary: 'Listen to Bill Mulligan talk about eBPF, Cilium and much more.'
```
Comment (xmulligan) [inline src/posts/2023-07-03-ama-bill/index.md:5]:
```suggestion
ogSummary: 'Listen to Bill Mulligan talk about eBPF, Cilium, and much more.'
```

File: src/posts/2023-07-03-cilium-empowering-k8s/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-07-03T11:28:00.000Z'
+title: 'Cilium: Empowering Kubernetes Networking and Security'
+ogImage: ogimage.png
+ogSummary: 'Dive into this step-by-step guide onCilium, and how it works'
```
Comment (xmulligan) [inline src/posts/2023-07-03-cilium-empowering-k8s/index.md:5]:
```suggestion
ogSummary: 'Dive into this step-by-step guide on Cilium, and how it works'
```

File: src/posts/2023-07-04-cilium-cni/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-07-04T11:28:00.000Z'
+title: 'Cilium CNI: A Comprehensive Deep Dive Guide for Networking and Security Enthusiasts!'
+ogImage: ogimage.png
+ogSummary: 'Dive into this step-by-step guide onCilium, and how it works'
```
Comment (xmulligan) [inline src/posts/2023-07-04-cilium-cni/index.md:5]: This is the same description as above

Comment (hacktivist123) [reply src/posts/2023-07-04-cilium-cni/index.md:5]: thanks for catching that, must have missed that.

**Review by xmulligan — APPROVED** [review]

**Pull request merged**